[2023-03-12T22:42:47.379+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: validacao_pu.task_calcular_pu_externo_dfg scheduled__2023-03-09T00:00:00+00:00 [queued]>
[2023-03-12T22:42:47.479+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: validacao_pu.task_calcular_pu_externo_dfg scheduled__2023-03-09T00:00:00+00:00 [queued]>
[2023-03-12T22:42:47.479+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-03-12T22:42:47.492+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 6
[2023-03-12T22:42:47.492+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-03-12T22:42:47.599+0000] {taskinstance.py:1300} INFO - Executing <Task(_PythonDecoratedOperator): task_calcular_pu_externo_dfg> on 2023-03-09 00:00:00+00:00
[2023-03-12T22:42:47.609+0000] {standard_task_runner.py:55} INFO - Started process 1613 to run task
[2023-03-12T22:42:47.624+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'validacao_pu', 'task_calcular_pu_externo_dfg', 'scheduled__2023-03-09T00:00:00+00:00', '--job-id', '166', '--raw', '--subdir', 'DAGS_FOLDER/examples/validacao_pu.py', '--cfg-path', '/tmp/tmpb3_qjog6']
[2023-03-12T22:42:47.630+0000] {standard_task_runner.py:83} INFO - Job 166: Subtask task_calcular_pu_externo_dfg
[2023-03-12T22:42:47.959+0000] {task_command.py:388} INFO - Running <TaskInstance: validacao_pu.task_calcular_pu_externo_dfg scheduled__2023-03-09T00:00:00+00:00 [running]> on host 40565fc59ef6
[2023-03-12T22:42:48.166+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=alexandre.yoshimatsu
AIRFLOW_CTX_DAG_ID=validacao_pu
AIRFLOW_CTX_TASK_ID=task_calcular_pu_externo_dfg
AIRFLOW_CTX_EXECUTION_DATE=2023-03-09T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-03-09T00:00:00+00:00
[2023-03-12T22:42:48.169+0000] {logging_mixin.py:137} INFO - Operação: dfg
[2023-03-12T22:42:48.169+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-03-12T22:42:48.196+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=validacao_pu, task_id=task_calcular_pu_externo_dfg, execution_date=20230309T000000, start_date=20230312T224247, end_date=20230312T224248
[2023-03-12T22:42:48.271+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-03-12T22:42:48.405+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
